### ISSUE TITLE
Performance Audit 2017: Investigate areas during plugin init() that could be deferred to later

### ISSUE THREAD

**Issue by nicjansma** [SOASTA/boomerang#167]
We should audit all plugins' `init()` to see if there's any work being done that could be deferred to later (e.g. beacon).

Work during `init()` is on the critical path to page load.  If there are operations we can wait on until beacon (after `onload`), we should.
